### PR TITLE
Update copy around architecture

### PR DESCRIPTION
**Pull request by tynes**
- Add new architecture diagram
- Generalize + update some of the wording

### CODE REVIEW

File: docs/bpanel-overview.md [hunk 1]
```diff
@@ -24,13 +24,20 @@ The possibilities of things that can be built with a BMS are potentially endless
...
-
-![bpanel architecture](/img/tech-diagram.png "bpanel architecture")
-
-This allows for a lot of flexibility. You could host your app server remotely, allowing for multiple users to access it from different clients and all talking to the same remote node. With bcoin's support for a separate wallet server, you can have a bPanel setup that is only meant to interact with a wallet server or only a node (Full, SPV, or Neutrino). You could use bcoin's built-in wallet services to manage wallets and accounts or handle key management locally in the clients. This also provides an extra layer of security by not giving users direct access to your node.
+The [default configuration](/docs/quick-start.html) for bPanel uses `docker-compose` to manage multiple docker containers.
+It will launch an app server that acts as a router and static file server, a TLS terminating nginx reverse proxy,
+a bcoin full node and a bcoin wallet server. Any of these services can be ran on their own - with the proper
+[configuration](/docs/quick-start.html#configuration), it is possible to use bPanel with a remote bitcoin node.
```
Comment (bucko13) [inline docs/bpanel-overview.md]: should probably be `with a remote bcoin node` as it's not really possible with other nodes that don't support the bcoin API (unless you only use RPC).

File: docs/quick-start.md [hunk 2]
```diff
@@ -27,41 +27,53 @@ To spin up your app server, a bcoin node on regtest, and mine 101 blocks (and ge
 This is primarily a setup for development purposes
 (though it could be used in production with some modification).
 
-To spin up your webapp, server, a bcoin node on regtest, and generate
-50 regtest BTC for your primary wallet, clone & navigate to this repo then:
+#### Running with docker
+
+To start the application and generate 50 regtest BTC for your primary wallet,
```
Comment (bucko13) [inline docs/quick-start.md]: this actually isn't necessarily true anymore since you need to have the right configs uncommented to generate the test coins. Maybe say the generate coins as an aside mentioning you need to enable the appropriate configs (and link to that section again here).

File: docs/quick-start.md [hunk 3]
```diff
@@ -27,41 +27,53 @@ To spin up your app server, a bcoin node on regtest, and mine 101 blocks (and ge
...
 
 ## Configuration
-Configurations are shared between the two docker containers.
 
-Your bcoin node will expect an API key given to it.
-If you are connecting to an existing node, you can set an API key
-by adding it to the `secrets.env` file and set `BCOIN_API_KEY=[YOUR-AWESOME-KEY]`.
-This key can be any value you want (but if you are running a node with real Bitcoins, make sure it's secure!).
-__NOTE: DO NOT CHECK THIS FILE IN TO VERSION CONTROL.__
+Configuration shared between the `app` and `bcoin` containers.
```
Comment (bucko13) [inline docs/quick-start.md]: Maybe rephrase to 
> This section covers configurations shared between the `app` and `bcoin` containers to allow them to communicate together.

File: docs/bpanel-overview.md [hunk 4]
```diff
@@ -24,13 +24,20 @@ The possibilities of things that can be built with a BMS are potentially endless
...
+(It is even possible to create a plugin that runs a bcoin node directly in the client).
+
+![bpanel architecture](/img/bpanel-architecture.png "bpanel architecture")
```
Comment (bucko13) [inline docs/bpanel-overview.md]: Do we want to replace the image that's there now entirely? Your diagram has specific comments talking about how TLS works, but the original one is meant as an introduction to people that have zero familiarity with bpanel. Might be worth either seeing if Darren can update the diagram, and/or having a separate section on this page about TLS (since it's optional anyway depending on the setup) and add your diagram there.

**Review by bucko13 — CHANGES_REQUESTED** [review]

File: docs/bpanel-overview.md (hunk 4 above)
Comment (tynes) [reply docs/bpanel-overview.md]: Just broke it out into its own part

File: docs/quick-start.md [hunk 5]
```diff
@@ -61,7 +60,12 @@ You can also use custom configs to connect to an existing bcoin node
 
 ## Configuration
 
-Configuration shared between the `app` and `bcoin` containers.
+This section covers the shared configuration between the `app`
+and `bcoin` containers necessary for their communication.
+
+The configuration is managed via environment variables set in the `bcoin.env` file.
+Secrets are managed in a `secrets.env` file which is listed in the `.gitignore`.
+__NOTE: DO NOT CHECK THIS FILE IN TO VERSION CONTROL.__
```
Comment (bucko13) [inline docs/quick-start.md]: Should be clear that this is about the secrets file and not bcoin.env. e.g. "NOTE: The secrets.env file has sensitive information and should not be checked into version control" I think there was a sentence about that before the bold part but it looks like that's been taken out now.

Comment (tynes) [reply docs/quick-start.md]: 👍

**Review by bucko13 — APPROVED** [review]
